hepdata/ext/elasticsearch/admin_view/api.py

```diff
@@ -160,32 +166,58 @@ def index_submission(self, submission):
 
         if record_information:
             collaboration = ','.join(record_information.get('collaborations', []))
-
-            self.add_to_index(_id=submission.publication_recid,
+            coordinator_request = CoordinatorRequest.query.filter_by(user=submission.coordinator).first()
```

Comment:
Maybe require `approved=True` in the filter in case the same user has other rejected Coordinator requests.